Test/test2/Rlimitouts0.bpl

```diff
@@ -0,0 +1,87 @@
+// RUN: %boogie -rlimit:8000 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
```

Comment:
OK. It is updated to use %OutputCheck. I also rename the functions to TestRlimit*, and made TestRlimit1 to verify so that it is easier to check.